h/static/scripts/render-markdown.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+var escapeHtml = require('escape-html');
+var katex = require('katex');
+var showdown = require('showdown');
+
+function targetBlank() {
+  function filter(text) {
+    return text.replace(/<a href=/g, '<a target="_blank" href=');
+  }
+  return [{type: 'output', filter: filter}];
+}
+
+var converter;
+
+function renderMarkdown(markdown) {
```

Comment:
It's fairly obvious but I think a docstring on this wouldn't hurt, to clarify the input and output. AIUI the input is markdown source text (not including KaTeX source text or HTML from KaTeX - those bits have already been replaced by placeholders) and the output is HTML.